frontend/svelte/env.config.mjs

```
@@ -7,6 +7,19 @@
  * The configuration is use in the rollup build but also in the parser of the static files - e.g. build.index.mjs to output the index.html with a CSP.
  */
 
+/**
+ * Returns the given environment variable, if defined and non-empty, else throws an error.
+ */
+function getRequiredEnvVar(key) {
+  const value = process.env[key];
+  if (undefined === value || value === "") {
+    throw new Error(`Environment variable '${key}' is undefined.`);
+  }
+  return value;
+}
```

Almost all the configuration is now read directly from the environment.  If a required value is missing, we panic rather than trying to guess.

Note:  I am getting the configuration from the environment due to tradition.  The data is also available in a JSON file, it might be easier to use that.  I don't mind either way.  PRs are welcome from anyone who finds that getting config from JSON is easier.

---

I don't have a strong opinion, but the possibility to override with env vars seems a good feature to me.

Maybe for easily test locally or whatever it's easier not not recreate the whole config json every time and use env vars.

Not sure, but I'd say we can leave it. If we stop using env vars, then I would go to remove the complexity.